collection_manager/collection_manager/services/CollectionWatcher.py

```diff
@@ -1,12 +1,12 @@
 import asyncio
+import time
 import logging
 import os
 from collections import defaultdict
 from typing import Dict, Callable, Set, Optional, Awaitable
 import yaml
 from watchdog.events import FileSystemEventHandler
-from watchdog.observers import Observer
-from yaml.scanner import ScannerError
+from watchdog.observers.polling import PollingObserver as Observer
```

Comment:
This line is very important as it tells the `watchdog` library to poll the filesystem instead of listening for events. Nothing else needed to change

Reply:
Perfect